crates/alerter/src/main.rs

```diff
@@ -278,6 +295,9 @@ struct Arguments {
 
     #[clap(long, env, default_value = "https://api.cow.fi/mainnet/")]
     orderbook_api: String,
+
+    #[clap(long, env, default_value = "9587")]
```

Comment:
nit: 
```suggestion
    #[clap(long, env, default_value = "9588")]
```

Small suggestion to use 9588 as 9586 and 9587 are used by the api and solvers respectively.